README.md

```diff
@@ -109,4 +109,4 @@ export FFF_TRASH=~/.cache/fff/trash
 
 ## Why?
 
-¯\\_(ツ)_/¯
+¯\\\_(ツ)_/¯
```

Comment:
```sh
¯\\_(ツ)_/¯
```

Reply:
The lack of arms is intended?

Reply:
Yes.

People don't like it and I keep getting PRs for it which I find funny.

Reply:
Oh, haha. It's not that I didn't like it, it's just a common mistake so I assumed that it wasn't intended. It does look goofier that way. :D